.github/workflows/release.yml

```diff
@@ -150,17 +155,24 @@ jobs:
         run: ci/cli.sh
       - name: install
         run: ci/install.sh
+        if: ${{ matrix.upgrade == false }}
+      - name: install last release
+        run: ci/install_last_release.sh
+        if: ${{ matrix.upgrade == true }}
       - name: wait
         run: sleep 600
-      - name: before downgrade test
-        run: ci/downgrade_setup.sh
-        if: ${{ matrix.downgrade == true }}
+      - name: before version upgrade/downgrade setup
+        run: ci/before_update_setup.sh
+        if: ${{ matrix.downgrade == true || matrix.upgrade == true}}
```

Comment:
Will this step run on both update and downgrade tests? Maybe renaming to `before_setup.sh`?

Reply:
yes it will run for both upgrade and downgrade ci test. 
other ci test will not use it so `before_setup.sh` looks a title bit more generic for that?
what do you think?

Reply:
yeah, I see. Maybe `before_additional_test.sh`?

Reply:
changed it